lib/reference_model/README.md

```diff
@@ -0,0 +1,20 @@
+# Reference Model
+
+The reference model is built as a shell around Spike, with the goal of properly simulating asynchronous events.
+
+Spike is responsible for the functional execution, while the pipeline shell is responsible for timing interrupts and other asynchronous events, and timing the RVFI output for comparison with the core.
+
+TODO: Add a more detailed explenation
+
+## Usage 
+
+To run the reference model alongside cv32e40s, run `make` as normal, but include the following:
+
+- set `USE_RM=YES` to enable the reference model.
+
+- set `SPIKE_HOME=<path to riscv/riscv-isa-sim>` to use another spike version.
+If not set, spike from `vendor/riscv/riscv-isa-sim` will be used.
+
+For example:
+`make test TEST=hello-world USE_RM=YES SPIKE_HOME=~/core-v-verif/vendor/riscv/riscv-isa-sim`
```

Comment:
When running `make comp USE_RM=1` I get
`uvmt_cv32e40s_tb.sv,1771|53): instance 'uvmt_cv32e40s_tb.reference_model' of design unit 'uvmt_cv32e40s_reference_model_wrap' is unresolved in 'worklib.uvmt_cv32e40s_tb:sv'.`

Is this a known problem with a known fix?

Reply:
I think I know the problem... One moment, I shall look for a fix.

Reply:
I will not fix it now. But for the future:
This should be ported to Xcelium, or rather could live in `mk/uvmt/uvmt.mk` instead of `mk/uvmt/vsim.mk` as much as possible.
But, I don't know about the `-dpicpppath` vsim option and how to port this etc, so I conclude that now is not the time to fix that.
(I will use vsim while testing.)